orders_postman_collection.json

```diff
@@ -1,8 +1,8 @@
 {
```

Comment:
Why are there so many deletions in this file? Is it intentional? If not, please remove these changes from this PR.

Reply:
So the thing is, I had imported the file to postman -> added the extra action request -> exported it.
Thus, I had to replace the original file which Git realizes as making individual changes to each line.